package.xml

```diff
@@ -11,16 +11,21 @@
 	<url type="repository">https://github.com/ethz-asl/libnabo</url>
 	<author email="stephane.magnenat@mavt.ethz.ch">Stéphane Magnenat</author>
 
-	<!-- The *_depend tags are used to specify dependencies -->
-	<buildtool_depend>cmake</buildtool_depend>
-	<build_depend>eigen</build_depend>
+	<!-- If a pure-cmake build is desired, the buildtool should be switched to 'cmake' -->
```

Comment:
Why do you suggest this option? Pure CMake builds do not need to use catkin at all.

Reply:
Okay, maybe it makes sense to keep it there.